src/demo.scss

```diff
@@ -20,23 +20,99 @@
   * defined in the project theme, and provided a handful of helpers which offer shorthand
   * ways of generating common CSS decorations.
   */
-@import 'FoundryCSS/base';
+@import "FoundryCSS/base";
 
 /*
   * FoundryCSS UI;
   *
   * Foundry's UI styles along with a default theme. The default theme provides
   * default values for any configuration not provided by the Project Theme.
   */
-@import 'FoundryCSS/ui/blocks/all';
-@import 'FoundryCSS/ui/components/all';
-@import 'FoundryCSS/ui/forms/all';
-@import 'FoundryCSS/ui/layout/all';
-@import 'FoundryCSS/tachyons/typography/all';
-@import 'FoundryCSS/tachyons/utilities/all';
+@import "FoundryCSS/ui/blocks/all";
+@import "FoundryCSS/ui/components/all";
+@import "FoundryCSS/ui/forms/all";
+@import "FoundryCSS/ui/layout/all";
+@import "FoundryCSS/tachyons/typography/all";
+@import "FoundryCSS/tachyons/utilities/all";
 
 /*
-  * Shame
-  * A temporary home for smelly CSS we will clean up later.
+  * Demo page styles
   */
-// @import 'shame';
+
```

Comment:
Should these not be in a file of their own?

Reply:
Possibly but the demo file never gets pulled through into a project anyway - I've just thrown it together to style up the markup around our examples. AKA I was being lazy - if everyone votes to tidy that away regardless I will do that.